lib/components/edit-program-modal/index.tsx

```diff
@@ -0,0 +1,27 @@
+import ProgramForm from '../program-form';
+import PopUp from '../pop-up';
+
+type EditProgramModalProps = {
+  modalManagement: {
+    isOpen: boolean;
+    onOpen: () => void;
+    onClose: () => void;
+  };
+};
+
+const EditProgramModal: React.FC<EditProgramModalProps> = ({ modalManagement }) => {
+  return (
+    <PopUp
+      header="Edit Event"
+      manageModal={modalManagement}
+      confirmText="Edit"
+      size="2xl"
```

Comment:
Will the size of this remain constant regardless of window size / screen resolution?

Reply:
yes, the modal is still responsive, the '2xl' just means it'll take up the majority of the screen even when resized